contents/blog/posthog-vs-launchdarkly.md

```diff
@@ -0,0 +1,632 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Calculate the recommended run time for your experiments</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+PostHog automatically calculates a recommended run time based on past data and minimally acceptable improvements. This helps you avoid the [peeking problem](/blog/ab-testing-mistakes) and end your experiment at the right time.
+
+![Run time calculator](../images/blog/posthog-vs-launchdarkly/runtime.png)
+
+### Pricing
+
+PostHog and LaunchDarkly price significantly differently. 
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td className="text-center"><strong>PostHog</strong></td>
+            <td className="text-center"><strong>LaunchDarkly</strong></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Pricing</strong></td>
+            <td className="text-center">Simple per request pricing</td>
+            <td className="text-center">Based on MAU + seats + add-ons</td>
+        </tr>
+        <tr>
+            <td><strong>Free plan</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center">Trial only</td>
+        </tr>
+        <tr>
+            <td><strong>Free experiments</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+        <tr>
+            <td><strong>Free collaboration</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+PostHog’s [feature flag pricing](/pricing?product=feature-flags) is pay-per-request (and A/B tests use feature flags). There is a generous free tier of 1M requests per month with all features, add-ons, and integrations available.
+
+![PostHog pricing](../images/blog/posthog-vs-launchdarkly/posthog-price.png)
+
+LaunchDarkly’s pricing is more complicated. They charge by seat starting at $10/seat/month. On top of this, they charge based on "monthly context instances" (MCIs) also known as monthly active users. The pro plan only gets 1,000 frontend MCIs per month. You must upgrade to the $20/seat/month to more than 1,000 MCIs, then pay more beyond 10,000.
+
+![LaunchDarkly pricing](../images/blog/posthog-vs-launchdarkly/launchdarkly-price.png)
+
+As mentioned earlier, many of the security, approval, and workflow features that differentiate them are only available at the opaque "Enterprise" level.
+
+#### Example scenarios
```

Comment:
Adjusted this section to look like this:

![Screen Shot 2023-10-03 at 10 18 15 AM](https://github.com/PostHog/posthog.com/assets/34755028/003246a9-12a9-4c1d-a1c9-cc37300af7db)

Happy to just remove it if its too confusing.

Reply:
I think this is fine. FYI, you've got some inconsistency on using `1000` instead of `1,000. Might be a US / British English thing, but I'd generally put a comma in any number 1,000 or more, just for consistency.